mineral-ui/mineral-ui
# [Menu] Add component

### Description
- Add Menu, MenuDivider, MenuHeading, and MenuItem components
- Add demo with example(s) for each
- Add snapshots for each

### Motivation and context

Connects #322 

### Screenshots or videos, if appropriate

<img width="328" alt="screen shot 2017-09-08 at 4 18 17 pm" src="https://user-images.githubusercontent.com/486540/30233208-5ecf742e-94b1-11e7-83bb-35a04fa5bcf0.png">

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to various Menu* components
3. Confirm necessary functionality and appearance

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist




### How does this PR make you feel?

![Menu](https://media.giphy.com/media/l4Jza5P4cMcCTHoGs/giphy.gif)
